paper.md

```diff
@@ -35,65 +35,75 @@ bibliography: paper.bib
+Such simple approach allows git-annex to scale to manage virtually arbitrarily large files, and "link" files in a Git repository to vast data resources available online. <!-- possibly state that number of files/objects remains limit imposed by Git -->
+
+## Why Git and git-annex alone are not enough
 
-## Why git and git-annex alone are not enough
+<!-- this is "what does datalad add to Git+git-annex -->
+<!-- MIH thinks: #1 nesting, #2 reproducible execution, #3 additional software adaptors for concrete services relevant for science -->
 
 **They are generic and might lack support for domain specific solutions.** 
-Git can interact with other repositories on the file system or accessible via a set of standard (ssh, http) or custom (git) network transport protocols.
-Interaction with non git-aware portals should then be implemented via custom git transfer protocols, as it e.g. was done in git-remote-rclone [TODOREF].
-Git-annex provides access to a wide range of external data storage resources via various protocols, but cannot implement all idiosyncracies of any individual data portal.
+Git can interact with other repositories on the file system or accessible via a set of standard (ssh, http) or custom (Git) network transport protocols.
+Interaction with non git-aware portals should then be implemented via custom Git transfer protocols, as it e.g. was done in git-remote-rclone [TODOREF] <!-- this is a largely incomplete/draft implementation, better ref OSF variant -->.
+Git-annex provides access to a wide range of external data storage resources via various protocols, but cannot implement all idiosyncracies of any individual data portal. <!-- not clear why that would be true. likely only due to lack of interest -->
 In particular, scientific data is frequently stored in compressed archives to reduce its storage demands, or on specialized servers, such as XNAT (www.xnat.org).
 To address these demands, git-annex implemented support for external special remotes [@git-annex:special_remotes_protocol].
-This allowed DataLad and many other projects to access and provide git-annex special remotes to facilitate access to an ever growing collection of resources [@git-annex:special_remotes] and to overcome technological limitations (e.g., maximal file sizes, or file system inodes limits).
+This allowed DataLad and many other projects to access and provide git-annex special remotes to facilitate access to an ever growing collection of resources [@git-annex:special_remotes] and to overcome technological limitations (e.g., maximal file sizes, or file system inodes limits). <!-- maybe phrase as "git-annex delegates..." or use the term "plugin" to make clear that a special remote hooks directly into git-annex and is a standardization of some arbitrary 3rd-party protocol -->
```

Comment:
I would prefer to not introduce additional "entities" ("delegates" and "plugin") since they would require additional description of them, and would be a bit "alien" since git-annex itself does not use those terms.  I have suggested wording to improve preceding sentence on that, and I think we can shorten this sentence a bit:

```suggestion
This allowed DataLad and many other projects to facilitate access to an ever growing collection of resources [@git-annex:special_remotes] and to overcome technological limitations (e.g., maximal file sizes, or file system inodes limits).
```